addons/website_sale/models/product_template.py

```diff
@@ -94,6 +94,14 @@ def _default_website_sequence(self):
         relation='product_public_category_product_template_rel',
     )
 
+    published_date = fields.Datetime(
```

Comment:
Yeah, also felt like this is more in line with `create_date` & `write_date`, just went with `published_date` because that's what they use in `social`, `website_blog`, and `website_hr_recruitment`.